Configuration/TCA/Overrides/tt_content.php

```diff
@@ -5,44 +5,43 @@
 
 defined('TYPO3') or die('Access denied.');
 
+// Add tt_content search group
+$GLOBALS['TCA']['tt_content']['columns']['CType']['config']['itemGroups']['search'] = 'LLL:EXT:solr/Resources/Private/Language/locallang.xlf:plugin_results';
```

Comment:
Should we use a more unique key like `tx_solr` for the item group?

Reply:
Internally discussed, we'll use "search".